src/main/java/org/myrobotlab/framework/Service.java

```diff
@@ -1405,7 +1341,7 @@ public ServiceConfig getConfig() {
   }
 
   public ServiceConfig load() throws IOException {
-    ServiceConfig config = Runtime.load(getName());
+    ServiceConfig config = Runtime.load(null, getName(), getClass().getSimpleName());
```

Comment:
it's confusing to pass null here..  just rely on the load signature that doesn't require the plan first.

Reply:
passing in the "plan" is going to go away - the parameter will be removed